tensorflow/workspace2.bzl

```diff
@@ -597,10 +597,11 @@ def _tf_repositories():
     )
 
     # WARNING: make sure ncteisen@ and vpai@ are cc-ed on any CL to change the below rule
+    # fix for https://github.com/protocolbuffers/upb/issues/300 - using grpc v1.33.2
     tf_http_archive(
         name = "com_github_grpc_grpc",
-        sha256 = "b956598d8cbe168b5ee717b5dafa56563eb5201a947856a6688bbeac9cac4e1f",
-        strip_prefix = "grpc-b54a5b338637f92bfcf4b0bc05e0f57a5fd8fadd",
+        sha256 = "a21c3a70eb356e0be2d989fdface89633421f2eddb95a1ede1f10a4804d7860b",
+        strip_prefix = "grpc-ee5b762f33a42170144834f5ab7efda9d76c480b",
         system_build_file = "//third_party/systemlibs:grpc.BUILD",
         patch_file = "//third_party/grpc:generate_cc_env_fix.patch",
```

Comment:
I meant checking this patch more then reverting version

Reply:
Something like `third_party/grpc/generate_cc_env_fix.patch`
```
@@ -141,6 +141,7 @@
         outputs = out_files,
         executable = ctx.executable._protoc,
         arguments = arguments,
+        use_default_shell_env = True,
     )
 
     return struct(files = depset(out_files))
```

Reply:
Also protobuf 3.15.2 was introduced with https://github.com/grpc/grpc/pull/25565. What is the minimal GRPC release version to support protobuf 3.15.2?

Reply:
Hi @bhack I am not completely sure what you meant by checking the patch. Could you please provide some details on how to do that? 
Also, the minimum grpc version to support protobuf v3.15.2 - Is there a location I can go lookup for such mapping?

Reply:
The patch is in `third_party/grpc/generate_cc_env_fix.patch` It needs to be compatible with the GRPC version that you want to use.

@mihaimaruseac On what GRPC release we could align? Internal tests require a specific version?

Reply:
Latest should be good, I think

Reply:
@satishpasumarthi So we could try with `grpc v1.36.4` and you need to change `third_party/grpc/generate_cc_env_fix.patch` with a `diff` against original `grpc v1.36.4` source.

Reply:
@bhack The `generate_cc.bzl` file is located at `bazel/generate_cc.bzl` in `grpc` repository and `third_party.grpc.generate_cc.bzl` is located at `thirdparty/systemlibs` of `tensorflow`.  I've updated the `generate_cc_env_fix.patch` file with the diff between the above two files. Also bumped grpc version to `v1.36.4` (latest)

Reply:
Probably I don't think that is required to touch `third_party/systemlibs` but I've not comipled the new version locally. 
You need just to check that we could apply @Flamefire patch cause it was not merged upstream https://github.com/grpc/grpc/pull/23664

Reply:
@bhack So do you mean the grpc/grpc#23664 changes should be in `generate_cc_env_fix.patch` ? 
If that is the case then the file `generate_cc_env_fix.patch`  doesn't need any change. 
I was under the impression that the `generate_cc_env_fix.patch` file was created by a diff between `systemlibs` and `generate_cc.bzl` in tensorflow. Do you want me to revert the changes to the file `generate_cc_env_fix.patch` then?

Reply:
Ok if the patch is compatible you can leave that as it is currently in master.

Reply:
reverted `generate_cc_env_fix.patch` to see if the tests pass.

Reply:
Yes, please don't remove the `default_shell_env` thing I added or otherwise we won't be able to compile TF on HPC systems (worldwide through EasyBuild) and need to repatch the patch for a bug that has a fix for almost a year now...